content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -356,4 +356,54 @@ CONTAINER ID        IMAGE               CREATED              STATE
 See [kubernetes-sigs/cri-tools](https://github.com/kubernetes-sigs/cri-tools)
 for more information.
 
-
+## Mapping from docker cli to crictl
+
+The exact versions for below mapping table are for docker cli v1.40 and crictl v1.19.0. Please note that the list is not exhaustive. For example, it doesn't include experimental commands of docker cli.
+
+Warn: the output format of CRICTL is similar to Docker CLI, despite some missing columns for some CLI. Make sure to check output for the specific command if your script output parsing.
+
+### Retrieve Debugging Information
+
+{{< table caption="mapping from docker cli to crictl - retrieve debugging information" >}}
+docker cli | crictl | Description | Unsupported Features
+-- | -- | -- | --
+attach | attach | Attach to a running container | `--detach-keys`, `--sig-proxy`
+exec | exec | Run a command in a running container | `--privileged`, `--user`, `--detach-keys`
+images | images | List images |  
+info | info | Display system-wide information |  
+inspect | inspectinspecti | Return low-level information on a container, image or task |  
+logs | logs | Fetch the logs of a container | `--details`
+ps | ps | List containers |  
+stats | stats | Display a live stream of container(s) resource usage statistics | Column: NET/BLOCK I/O, PIDs
+version | version | Show the runtime (Docker, ContainerD, or others) version information |  
```

Comment:
Aside: the first two columns could go in backticks

Reply:
Sure. Let me update.